test/electron/features/renderer-config.feature

```diff
@@ -16,3 +16,25 @@ Feature: Setting config options in renderers
             | property | config                      |
             | appType  | { "appType": "real great" } |
             | codeBundleId  | { "codeBundleId": "1.0.0-r0123" } |
+            | user  | { "user": { "id": "3", "name": "Bugs Nag", "email": "bugs.nag@bugsnag.com" } } |
+            | context  | { "context": "renderer context" } |
+            | metadata  | { "metadata": { "renderer": { "key": "value" } } } |
+
+    Scenario Outline: Clearing config options set in renderer config
```

Comment:
Oh sorry, that test is checking that those values do exist - I noticed those config options were missing from that existing scenario outline so I added them in. Maybe I should have left that out as it's not directly related to this fix

Reply:
But I think this does apply to the `clear-context` test in the new scenario as I think that will revert to a default context if it's cleared. Just trying to figure out what that would be...

Reply:
Yeah, that's what I'm commenting on — the "Clearing config options set in renderer config" scenario

- `clear-user.json` has a `user` object with an ID, but it will still pass if there's an `email` & `name`
- `clear-context.json` doesn't have a `context` key, so it will pass regardless of the value
- `clear-metadata.json` has a `metadata` object with some automatically collected stuff in, but it will still pass if there's a `renderer` section

Reply:
Aah yes I see what you mean. Updated.